README.md

```diff
@@ -3,13 +3,40 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  1. Asympotic analysis ignored the constant factor and lower-order terms, in the
+  real world practical scenarios with finite input, these factos can impact the
+  performance significantly.
+  2. Asympotic analysis is a theoretical model, it does not account for the real-world
+  hardware, such as CP cache size, memory bandwith, diskI/O...These can also impact
+  the actual performance.
+  3. Asympotic analysis is often related with the size of the input. But in real world
```

Comment:
But we can do that -- we can analyze the complexity of performing a particular operation on a particular data structure.